### ISSUE TITLE
Pretrained  weight of RoadNet

### ISSUE THREAD

**Issue by metaism-ding** [yhlleo/DeepSegmentor#11]
Hi, there
You provided roughly trained weight in this repo, but the performance was not so good.
To get the accuracy mentioned in your paper, can you provide the well trained model of RoadNet?

Thanks

**Comment by yhlleo** [comment]
Yes, I moved the tensorflow version in the paper to pytorch version, but I didn't spend much time on training the model. I would like to provide the well-trained model as soon as possible. Thanks for you suggestion.

**Comment by metaism-ding** [comment]
Hi, Liu

By the way, can you tell me more details about collecting satellite imagery?
Currently, I am doing similar work using satellite data in Japan to recognize both the centerline and width.
1. You said in the paper than your data were from Google Earth, how did you downloaded it? I got my data from Google map with certain API.
2. How did you actually label the data? Directly mark the point of edge and make a line?
Google provide roadmap or terrain type data, can i utilize it to reduce the preprocess work?
3. Why you select discrete areas in Ottawa? What's the main difference of these train/valid_test areas?

Thank you for reading my questions.
Hope your answer.

**Comment by yhlleo** [comment]
Sorry for the late reply.

- I downloaded the images through a paid software, I'm not sure you can download the same level images from google api freely;
- To label the data, I did it manually by drawing points and connecting the neighbor points with straight lines. For the roadmap or terrain data, I'm not sure that they can be directly applied to extract edges and centerlines.
- We observed that there are few occlusions and shadows in CasNet dataset, so we chose images from Ottawa. Both train and test areas are surrounding Ottawa. The main difference is that they cover different areas.
